### ISSUE TITLE
Cross-Origin-Embedder-Policy

### ISSUE THREAD

**Issue by SavAct** [noname0310/babylon-mmd-vite-template#1]
The Vite server headers require "Cross-Origin-Embedder-Policy": "require-corp" to work with `babylon-mmd`. 
```ts
server: {
        headers: {
            "Cross-Origin-Opener-Policy": "same-origin",
            "Cross-Origin-Embedder-Policy": "require-corp"
        }
}
```

Otherwise, an exception for `SharedArrayBuffer` is thrown. However, I require "cross-origin" to embed different external resources. Can you suggest any solution to use babylon-mmd with a "cross-origin" setup?

**Comment by noname0310** [comment]
It is not possible to set headers with `cross-origin` and multithreading based on `SharedArrayBuffer` at the same time.

I'm using a proxy server in production to fetch external resources.

The problem is also solved by using a version without `SharedArrayBuffer`, which is `MmdWasmInstanceSPR`, but it doesn't support multithreading.

**Comment by SavAct** [comment]
Thank you for the fast reply. You have confirmed my concerns. I suggest the non-wasm instance is probably more performant than the single-threaded wasm instance on mobile devices that have a lot of low-energy cores.

**Comment by noname0310** [comment]
WASM implementations will always be faster than JS. The WASM runtime is designed to keep WASM interop costs extremely low and using SIMD instructions aggressively.

**Comment by SavAct** [comment]
Good to know, but by changing `MmdWasmInstanceMPR` to `MmdWasmInstanceSPR` the function [model.morph.resetMorphWeights();](https://github.com/noname0310/babylon-mmd-viewer/blob/1cf697c203b824c1537f19c8a4696ae054796cb8/src/Viewer/viewerUi.ts#L345) throws 
_Uncaught (in promise) TypeError: Cannot perform %TypedArray%.prototype.fill on a detached ArrayBuffer_
while loading a motion file after loading a model file. Do I have to miss this function in SPR mode?

**Comment by noname0310** [comment]
Fixed that issue and just released babylon-mmd 0.57.0. Please try this and let me know if you get the same issue.

**Comment by SavAct** [comment]
Thx, it works now
